src/js/components/BreadcrumbComponent.jsx

```diff
@@ -12,6 +12,12 @@ var BreadcrumbComponent = React.createClass({
     task: React.PropTypes.string
   },
 
+  shouldComponentUpdate: function (nextProps) {
+    return nextProps.app !== this.props.app
+      && nextProps.group !== this.props.group
```

Comment:
Cool!
But I am confused, shouldn't this be an OR-chain instead of an AND-chain?